### PR TITLE
Update the bioconductor skeleton

### PR DESCRIPTION
**Pull request by dpryan79**
This should do the following (mostly layed out in #588 ):

 - [x] Include a `license_file` if possible
 - [x] Recipes finally pass linting
 - [x] Detect as much as possible if packages require X
 - [x] Convert any SystemRequirements to actual conda requirements, if possible

### CODE REVIEW

**Comment by dpryan79** [comment]
It would of course be much nicer to just parse the system requirements like we do imports/depends/etc., but it turns out that this is completely unstructured. Given that, since we have to special case quite a few packages, we might as well just do all of them.

**Comment by dpryan79** [comment]
The only failing lints are occasional "long summary" message, which isn't a big deal.

File: bioconda_utils/cli.py [hunk 1]
```diff
@@ -681,13 +681,15 @@ def bioconductor_skeleton(
         if not bioc_version:
             bioc_version = _bioconductor_skeleton.latest_bioconductor_release_version()
         packages = _bioconductor_skeleton.fetchPackages(bioc_version)
+        needs_x = _bioconductor_skeleton.packagesNeedingX(packages)
+        print("need X: {}".format(needs_x))
```
Comment (bgruening) [inline bioconda_utils/cli.py]: remove?

Comment (dpryan79) [reply bioconda_utils/cli.py]: Oops, yes, old debug code

**Pull request merged**